src/amo/components/SearchForm.scss

```diff
@@ -0,0 +1,36 @@
+@import "~core/css/inc/vars";
+
+.SearchForm-form {
+  background: rgba(255, 255, 255, 0.2);
+  border: 0;
+  display: flex;
+
+  .SearchForm-query {
+    background: url('~amo/img/icons/magnifying-glass.svg') 18px 50% no-repeat;
+    border: 0;
+    color: $header-font-color;
+    font-size: $font-size-m;
+    flex-grow: 1;
+    height: 3rem;
+    margin: 0 auto;
+    max-width: 320px;
```

Comment:
Gosh, I can't remember, but you're right. I've opened #1151 to address it and some other comments.